example/lib/radar_chart/samples/radar_chart_sample1.dart

```diff
@@ -0,0 +1,228 @@
+import 'package:fl_chart/fl_chart.dart';
+import 'package:flutter/material.dart';
+
+const gridColor = Color(0xff68739f);
+const titleColor = Color(0xff8c95db);
+const fashionColor = Color(0xffe15665);
+const artColor = Color(0xff63e7e5);
+const boxingColor = Color(0xff83dea7);
+const entertainmentColor = Colors.white70;
+const offRoadColor = Color(0xFFFFF59D);
+
+class RadarChartSample extends StatefulWidget {
```

Comment:
Rename it to RadarChartSample1, (we need to mention the index number because we are going to implement more samples). Also, it keeps consistency along with other examples.